src/CatalogManager.php

```diff
@@ -67,8 +67,8 @@ protected function sanitizeRequestParams($request)
 		})->toArray();
 	}
 
-	protected function sanitize($value): string
+	protected function sanitize(string $value): string
 	{
-		return is_string($value) ? filter_var($value, FILTER_SANITIZE_STRING) : $value;
+		return stripslashes($value);
```

Comment:
Not needed to filter or escape attributes, Laravel is doing that on the request object and automatically is even adding slashes in quotes

Reply:
I removed the condition because the function signature says that always return an string